### PR TITLE
Jon/disp img

### PR DESCRIPTION
**Pull request by jonathonross**

### CODE REVIEW

File: scripts/user_label_image.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
+"""
+███████╗███╗   ██╗ █████╗ ██████╗  ██████╗ █████╗ ████████╗
```
Comment (jonathonpross) [inline scripts/user_label_image.py]: I'm pretty sure this is the most valuable thing I've added to this project

Comment (beardedtrogon) [reply scripts/user_label_image.py]: Consider a career in graphic design

**Comment by beardedtrogon** [comment]
Images are too big, can't see the arrows at the bottom. Any chance they can be resized to fit the screen?

**Comment by beardedtrogon** [comment]
@jonathonpross Command prompt hangs when I close the program with the X at the top right corner

**Comment by jonathonross** [comment]
while cv2.getWindowProperty('window-name', 0) >= 0:
    keyCode = cv2.waitKey(50)

Looks like there's a way to check, will need to wrap the function.

**Comment by jonathonpross** [comment]
@beardedtrogon rebased

**Comment by jonathonpross** [comment]
@beardedtrogon 

See if this works the way you want it now.

I am not sure exactly the path we want to take with classifying the bursts, but this will let you at least pull out the blobs that are cats that you want.

I think the next step is getting a better classifier (getting training data sorted)

**Pull request merged**